src/AuthLDAP.php

```diff
@@ -630,108 +527,16 @@ public function showForm($ID, array $options = [])
-            Dropdown::showNumber('ldap_maxlimit', ['value' => $this->fields['ldap_maxlimit'],
-                'min'   => 100,
-                'max'   => 999999,
-                'step'  => 100,
-                'toadd' => [0 => __('Unlimited')]
-            ]);
-            echo "</td><td colspan='2'></td></tr>";
-        } else {
-            $hidden .= "<input type='hidden' name='can_support_pagesize' value='0'>";
-            $hidden .= "<input type='hidden' name='pagesize' value='0'>";
-            $hidden .= "<input type='hidden' name='ldap_maxlimit' value='0'>";
-        }
-
-        echo "<tr class='tab_bg_1'>";
-        echo "<td>" . __('How LDAP aliases should be handled') . "</td><td colspan='4'>";
-        $alias_options = [
-            LDAP_DEREF_NEVER     => __('Never dereferenced (default)'),
-            LDAP_DEREF_ALWAYS    => __('Always dereferenced'),
-            LDAP_DEREF_SEARCHING => __('Dereferenced during the search (but not when locating)'),
-            LDAP_DEREF_FINDING   => __('Dereferenced when locating (not during the search)'),
-        ];
-        Dropdown::showFromArray(
-            "deref_option",
-            $alias_options,
-            ['value' => $this->fields["deref_option"]]
-        );
-        echo"</td></tr>";
-
-        echo "<tr class='tab_bg_1'>";
-        echo "<td>" . __('Domain name used by inventory tool for link the user') . "</td>";
-        echo "<td colspan='3'>";
-        echo Html::input('inventory_domain', ['value' => $this->fields['inventory_domain'], 'size' => 100]);
-        echo "</td></tr>";
-
-        echo "<tr class='tab_bg_1'>";
-        echo "<td>" . __('TLS Certfile') . "</td><td>";
-        echo "<input type='text' name='tls_certfile' class='form-control' id='tls_certfile' value='" . $this->fields["tls_certfile"] . "'>";
-        echo "</td>";
-        echo "<td>" . __('TLS Keyfile') . "</td><td>";
-        echo "<input type='text' name='tls_keyfile' class='form-control' id='tls_keyfile' value='" . $this->fields["tls_keyfile"] . "'>";
-        echo "</td>";
-        echo "</tr>";
-
-        echo "<tr class='tab_bg_1'>";
-        echo "<td>" . __('TLS Version') . "</td><td>";
-        Dropdown::showFromArray('tls_version', array_merge(['' => Dropdown::EMPTY_VALUE], self::TLS_VERSIONS), ['value' => $this->fields["tls_version"]]);
-        echo "</td>";
-        echo "<td><label for='timeout'>" . __('Timeout') . "</label></td>";
-        echo "<td>";
-        Dropdown::showNumber('timeout', ['value'  => $this->fields["timeout"],
-            'min'    => 1,
-            'max'    => 30,
-            'step'   => 1,
-            'toadd'  => [0 => __('No timeout')]
+        TemplateRenderer::getInstance()->display('pages/setup/ldap/adv_info.html.twig', [
+            'item' => $this,
+            'page_size_available' => self::isLdapPageSizeAvailable(false, false),
+            'ldap_start_tls_exists' => function_exists("ldap_start_tls"),
```

Comment:
This check was introduced in b99fca3257ddc41d6c09d69b698999ec0704bcd3 (Dec 27, 2006). This method is supposed to be available since PHP 4.2, and I do not see any documentation that says that it may be missing when the LDAP extension is installed.

IMHO, we can remove this condition and consider that thismethod is always available, as long as the LDAP extension is enabled..